README.md

```diff
@@ -0,0 +1,245 @@
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://render.com/"><img src="https://img.shields.io/badge/Render-46E3B7?style=for-the-badge&logo=render&logoColor=white"/></a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/"><img src="https://img.shields.io/badge/PostgreSQL-316192?style=for-the-badge&logo=postgresql&logoColor=white"/></a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Authentication</summary>
+  <ul>
+    <li><a href="https://jwt.io/"><img src="https://img.shields.io/badge/JWT-000000?style=for-the-badge&logo=JSON%20web%20tokens&logoColor=white"/></a></li>
+  </ul>
+</details>
+<br/>
+
+<!-- ERD -->
+
+### Entity Relationship Database <a name="ERD"></a>
+
+<br/>
+<img src="./readme_assets/wheels-and-deals-erd.jpeg" style="width: 400px">
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Authentication Endpoints: POST
+- Users Endpoints: GET, POST
+- Cars Endpoints: GET, POST, DELETE
+- Reservations Endpoints: GET, POST, DELETE
+- Documentation available [here](https://wheels-and-deals.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📅 Project's Kanban Board <a name="kanban-board"></a>
+
+Kanban Board available [here](https://github.com/users/Peter1907/projects/3/views/1) 
+
+The team of developers involved in the creation of this project is made up of 4 [authors](#authors).
+
+<img src="./readme_assets/kanban-board.png" style="width: 800px">
+
+<br/>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://wheels-and-deals.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📷 Documentation Screenshots <a name="documentation-ss"></a>
+
+  <img src="./readme_assets/documentation01_wheels_and_deals.png" style="width: 800px">
+  <img src="./readme_assets/documentation02_wheels_and_deals.png" style="width: 800px">
+
+<br/>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+   gem install rails
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Peter1907/wheels-and-deals-api.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd wheels-and-deals-api
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+```sh
+  bin/rails spec
+```
```

Comment:
- [ ] When  I run your tests according to instructions, this happens:

![image](https://user-images.githubusercontent.com/57421823/224192518-a2af662f-a456-4c66-bb5e-047fc2fd7551.png)

This is the reason:

![image](https://user-images.githubusercontent.com/57421823/224192563-cf838f04-c77a-4d29-b5bf-509efc71c1cc.png)

You are not pushing the master key to the repo.

Kindly, either push your master.key file (remove from gitignore) or include specific instructions on your readme about how to generate one.

These are the instructions:

```
1. Remove config/master.key and config/credentials.yml.enc
2. Run EDITOR=code rails credentials:edit in the terminal. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://user-images.githubusercontent.com/57421823/224192902-43c5981e-e13b-4906-8eb4-458f710ce773.png)